app/src/main/java/io/pumpkinz/pumpkinreader/NewsListFragment.java

```diff
@@ -98,9 +103,10 @@ public void onError(Throwable e) {
             }
 
             @Override
-            public void onNext(List<ItemPOJO> itemPOJOs) {
-                Log.d("stories", String.valueOf(itemPOJOs.size()));
-//                dataset.addAll(itemPOJOs);
+            public void onNext(List<News> items) {
+                Log.d("stories", String.valueOf(items.size()));
+                Log.d("stories content", items.toString());
+                dataset.addAll(items);
```

Comment:
Move observable to NewsAdapter alltogether.
NewsListFragment doesn't need to know about List<News> dataset at all.